advanced_guides/prefix.md

```diff
@@ -26,9 +26,9 @@ but not
 - `movies`
 - `musical`
 
-Even if, as you can observe, there is an `s` in `movies` and `musical`. The prefix search begins searching at the **start** of the word.
+Even if, as you can observe, there is an `s` in `movies` and `musical`. The prefix search begins searching at **the start of the word**. Notice that a prefix search is only done for the last word of a query, other words must be of the same length but can contain typos.
 
-When the is search `sho` only the documents containing `show` will be returned.
+When we search for `sho` only the documents containing `show` will be returned. Obviously we will also return words if you wrote a prefix with typos, you can [read more about the typo rules on the dedicated page](/advanced_guides/typotolerance.md#typo-tolerance-rules).
```

Comment:
```suggestion
If the query is `sho`, only the documents containing `show` will be returned. MeiliSearch also handles typos while performing the prefix search, you can [read more about the typo rules on the dedicated page](/advanced_guides/typotolerance.md#typo-tolerance-rules).```

Reply:
That’s better, yes.
Why did you prepend triple backticks at the end of the sentence though?

Reply:
Oh it was a GitHub bug when closing suggestion block on the same line as the code